src/code/utils/translate.js

```diff
@@ -17,6 +17,7 @@ import ja from './lang/ja'
 import nb from './lang/nb'
 import nn from './lang/nn'
 import tr from './lang/tr'
+import th from './lang/th'
```

Comment:
It appears that this list was also alphabetized, so bump this up a line.

Reply:
I have bumped it up a line.